lib/marked-renderer.js

```diff
@@ -0,0 +1,13 @@
+// A custom renderer that avoids wrapping <img> tags in <p> tags
+const { marked } = require('marked')
+
+class DesignSystemRenderer extends marked.Renderer {
+  paragraph (text) {
+    if (text.trim().startsWith('<img')) {
+      return `${text}\n`
```

Comment:
Do we need to worry about any other image related elements?

```js
return ['img', 'picture', 'figure'].some((tag) =>
  text.trim().startsWith(`<${tag}`)
)
  ? super.html(text)
  : super.paragraph(text)
```

Reply:
It's a good thought, but this really only covers instances where we use markdown image syntax. If we're inserting our own HTML with the other tags, it should be interpreted as HTML rather than a paragraph, and not use this renderer at all.

Given that `metalsmith-tag-cleaner` [only cared about `<img>` tags](https://github.com/justb3a/metalsmith-tagcleaner/blob/15479f123b173cbb10fddb902ef7f775d4aa1daa/lib/index.js#L42), and Github Flavoured Markdown only uses `<img>` when complied from image syntax, I'm inclined to leave it like that for now.

Reply:
Cheers @domoscargin, was worth a check! Thanks

Reply:
Thanks for reminding me that there was a neat way to have only one return, btw!